src/sketch.h

```diff
@@ -204,7 +204,8 @@ class Group {
         //WORKPLANE_BY_FACE          = 6004,
         // For extrudes, translates, and rotates
         ONE_SIDED                  = 7000,
-        TWO_SIDED                  = 7001
+        TWO_SIDED                  = 7001,
+        ONE_SKEWED                 = 7004
```

Comment:
Very cunning of you to introduce a new `Group::Subtype` in this way the saving is all taken care of. I like it :-)

But `TWO_SKEWED` is definitely required (as you already suggested).